src/tests/test_task3.py

```diff
@@ -0,0 +1,115 @@
+import ddt
```

Comment:
in fact too complex for test and test cases look pretty weird. I don't understand why is it tests only counts

in fact we can generate similar files for tests and check how would it handle it